File: cpp/leetcode/119.cpp
class Solution {
public:
    vector<int> getRow(int rowIndex) {
        if(rowIndex < 0) return {};
        vector<int> res(rowIndex + 1);
        for(int i = 0; i <= rowIndex; i++){
            for(int j = i; j >= 0; --j){
                if(j == 0 || j == i) res[j] = 1;
                else res[j] += res[j-1];
            }
        }
        return res;
    }
};

File: python/leetcode/119.py
class Solution(object):
    def getRow(self, rowIndex):
        A = [0] *(rowIndex + 1)
        A[0] = 1;
        for i in range(1, rowIndex + 1):
            for j in range(i, 0, -1):
                A[j] += A[j-1]
        return A